redirect.ejs

```diff
@@ -19,6 +19,9 @@
         var protocol = '<%- https ? "https" : "http" %>'
         var port = '<%- port %>'
         var platform = device.platform.toLowerCase()
+        if(platform === 'amazon-fireos') {
```

Comment:
Please put a space before the parentheses.
And I think we should add some comments to explain why this branch is necessary.

Reply:
Added, I hope it's not too verbose!